Simplenote/Resources/AppStoreStrings.pot

```diff
@@ -15,12 +15,14 @@ msgstr ""
 
 #. translators: App name to be displayed in the Apple App Store. Limit to 30 characters including spaces and commas!
 msgctxt "app_store_title"
-msgid "Simplenote – Notes and Todos"
+msgid "Simplenote – Notes and Todos
```

Comment:
This _may_ break the ability to import this file into GlotPress? Might be worth testing it with https://github.com/wordpress-mobile/docker-glotpress-linter?

Reply:
I didn't test what the linter said. I just went ahead and fixed it in [`0a9932f` (#1435)](https://github.com/Automattic/simplenote-ios/pull/1435/commits/0a9932f73d5fb5b31d0aa4d347087dcd9acd10b9)